packages/mehari-eio/mehari-eio.0.2/opam

```diff
@@ -1,17 +1,21 @@
 # This file is generated by dune, edit dune-project instead
```

Comment:
Why is `mehari-eio-unix.0.1` being remove? Removal of packages are not accepted in opam-repository except for extreme cases. If it still works fine, could you please add the `flags: deprecated` instead if it's a deprecated package?

Reply:
Oh right sorry, I didn't know that. It is corrected now!